Application/EdFi.Ods.Standard/Artifacts/PgSql/Structure/Ods/Changes/0120-CreateIdColumnUniqueIndexesForChangesTables.sql

```diff
@@ -0,0 +1,2 @@
+CREATE UNIQUE INDEX IF NOT EXISTS UX_6ad27b_Id ON changes.Snapshot(Id);
```

Comment:
Again fix the names of the scripts per above

Reply:
renamed